lib/http-proxy/passes/web-incoming.js

```diff
@@ -18,6 +18,11 @@ var nativeAgents = { http: httpNative, https: httpsNative };
  * flexible.
  */
 
+// 'aborted' event stopped working reliably on v15.5.0 and was later removed entirely
+var hasAborted = (function () {
```

Comment:
Super minor suggestion but when I first glanced at this I presumed it was a function for checking if a connection had been aborted. So maybe something like "supportsAbortedEvent" might be a bit of a clearer name?